Core/Python/get_device_list.py

```diff
@@ -55,61 +52,76 @@ def __init__(self, session_input, output_details):
             return
 
         self.__base_uri = 'https://%s' % self.__session_input["ip"]
-        self.__headers = {'content-type': 'application/json'}
 
         try:
-            if self.__authenticate_with_ome() is False:
+            AUTH_SUCCESS, HEADERS = self.__authenticate_with_ome()
+            print(HEADERS)
+            if not AUTH_SUCCESS:
+                print("Unable to authenticate with OME .. Check IP/Username/Pwd")
                 return
-        except requests.exceptions.RequestException as auth_ex:
+        except OSError:
```

Comment:
Would OSError map to Exceptions from the Requests library? I assumed they were returned on system call failures - wouldn't it be better to map to Exception instead?

Reply:
Done